datadog/resource_datadog_synthetics_test_.go

```diff
@@ -479,6 +479,27 @@ func syntheticsAPIAssertion() *schema.Schema {
 					Type:        schema.TypeString,
 					Optional:    true,
 				},
+				"targetjsonschema": {
+					Description: "Expected structure if `operator` is `validatesJSONSchema`. Exactly one nested block is allowed with the structure below.",
+					Type:        schema.TypeList,
+					Optional:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"jsonschema": {
+								Description: "The Json Schema to validate the body against.",
```

Comment:
We use full uppercase `JSON` in other places, and it aligns with the [JSON Schema spec](https://json-schema.org/specification)
```suggestion
								Description: "The JSON Schema to validate the body against.",
```